errors.go

```diff
@@ -22,6 +22,9 @@ import (
 
 // common error objects used for argument checking
 var (
+	ErrNeedPod         = errors.New("Pod must be specified")
 	ErrNeedPodID       = errors.New("Pod ID cannot be empty")
 	ErrNeedContainerID = errors.New("Container ID cannot be empty")
+	ErrNeedFile        = errors.New("file cannot be empty")
```

Comment:
actually, it seems usual to have errors starting with a lower case character in golang, so the other ones should be changed if any (a separate commit though). For reference I grepped through github.com/golang/go:

```
$ git grep -p 'errors.New.*"[A-Z]' src/ | wc -l
96
$ git grep -p 'errors.New.*"[a-z]' src/ | wc -l
1364
```